server/db/models/conversation.js

```diff
@@ -1,11 +1,18 @@
-const { Op } = require("sequelize");
+const { Sequelize, Op } = require("sequelize");
 const db = require("../db");
 const Message = require("./message");
 
-const Conversation = db.define("conversation", {});
+const Conversation = db.define("conversation", {
+  // Model attributes are defined here
+  user1UnreadMsg: {
+    type: Sequelize.INTEGER,
+  },
+  user2UnreadMsg: {
+    type: Sequelize.INTEGER
+  }
+});
```

Comment:
While this works as an implementation, it has the disadvantage of not being very scalable. If, in the future, we wanted to implement the ability to have more than 2 users in the conversation, this model would quickly outgrow our ability to expand it.

A more atomic and scalable implementation would be to mark each individual message as read or unread.

Please note that **you don't need to implement this suggestion**, since it would mean refactoring the hole implementation.

I would also suggest a different name for the properties, since the names are not descriptive of what they represent. If it's the number of unread messages, I would specify that it's a number in the name, for example `user1UnreadCount`.